packages/canary-integration/src/__tests__/model-instance-document.test.ts

```diff
@@ -2,20 +2,81 @@ import { jest } from '@jest/globals'
 import getPort from 'get-port'
 import { AnchorStatus, CeramicApi, CommitType, IpfsApi } from '@ceramicnetwork/common'
 import { createIPFS } from '@ceramicnetwork/ipfs-daemon'
-import { ModelInstanceDocument, ModelInstanceDocumentMetadata } from '@ceramicnetwork/stream-model-instance'
+import {
+  ModelInstanceDocument,
+  ModelInstanceDocumentMetadata,
+} from '@ceramicnetwork/stream-model-instance'
 import { createCeramic } from '../create-ceramic.js'
 import { anchorUpdate } from '@ceramicnetwork/core/lib/state-management/__tests__/anchor-update'
-import { Ceramic } from '@ceramicnetwork/core'
+import { Ceramic, CeramicConfig } from '@ceramicnetwork/core'
 import { CeramicDaemon, DaemonConfig } from '@ceramicnetwork/cli'
 import { CeramicClient } from '@ceramicnetwork/http-client'
 import { StreamID } from '@ceramicnetwork/streamid'
 import first from 'it-first'
 import { Model, ModelAccountRelation, ModelDefinition } from '@ceramicnetwork/stream-model'
+import { SqliteIndexApi } from '@ceramicnetwork/core/lib/indexing/sqlite/sqlite-index-api'
+import { listMidTables } from '@ceramicnetwork/core/lib/indexing/sqlite/init-tables'
+import knex, { Knex } from 'knex'
+import tmp from 'tmp-promise'
+
+const INDEXING_MODELS = [
+  'kjzl6cwe1jw147dvq16zluojmraqvwdmbh61dx9e0c59i344lcrsgqfohexp60s',
+  'kjzl6hvfrbw6c8c8ks6ndp7elnd03csx3q7934qmacryhqoqn1para6sfpau6xi',
+  'kjzl6cwe1jw145m7jxh4jpa6iw1ps3jcjordpo81e0w04krcpz8knxvg5ygiabd',
+]
+const FAKE_STREAM_ID = StreamID.fromString(
+  'kjzl6cwe1jw147dvq16zluojmraqvwdmbh61dx9e0c59i344lcrsgqfohexp60s'
+)
 
 const CONTENT0 = { myData: 0 }
 const CONTENT1 = { myData: 1 }
 const CONTENT2 = { myData: 2 }
 const CONTENT3 = { myData: 3 }
+const METADATA = { model: FAKE_STREAM_ID }
+
+let tmpFolder: tmp.DirectoryResult
+let dbConnection: Knex
+
+beforeEach(async () => {
+  tmpFolder = await tmp.dir({ unsafeCleanup: true })
+  const filename = `${tmpFolder.path}/tmp-ceramic.sqlite`
+  dbConnection = knex({
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+      filename: filename,
+    },
+  })
+})
+
+afterEach(async () => {
+  await dbConnection.destroy()
+  await tmpFolder.cleanup()
+})
+
+describe('init', () => {
```

Comment:
Does it really belong here? It looks like an existing test suite for sqlite-index-api.